### PR TITLE
Sabgit

### PR DESCRIPTION
**Pull request by sabgit**
Updates for PR review comments, bump up the version

### CODE REVIEW

File: installer.js [hunk 1]
```diff
@@ -90,15 +90,15 @@ function driverVersionString(driverName, targetPath)
...
+        throw new Error(`no driver exists with the name ${driverName}.`);
     }
 }
 
 function driverInstaller(browserName, browserVersion, targetPath)
 {
     if (typeof browserName !== 'string' || typeof browserVersion !== 'string' || typeof targetPath !== 'string')
     {
-        throw new Error('Parameters are not valid strings.');
+        throw new Error('parameters are not valid strings.');
```
Comment (thehcma) [inline installer.js]: the <blah> parameters are not valid strings

remove the period

File: installer.js [hunk 2]
```diff
@@ -123,7 +123,7 @@ function driverInstaller(browserName, browserVersion, targetPath)
     else
     {
         throw new Error(
-            `Browser name "${browserName}" is not a valid browser name. Valid browser names are: ${(VALID_BROWSER_NAMES).join(', ')}`
+            `browser name "${browserName}" is not a valid browser name. Valid browser names are: ${(VALID_BROWSER_NAMES).join(', ')}`
```
Comment (thehcma) [inline installer.js]: <blah> is not a valid browser name. The valid names are: <blah>

File: installer.js [hunk 3]
```diff
@@ -132,7 +132,7 @@ function driverInstaller(browserName, browserVersion, targetPath)
     if (browserVersion && !browserDriverVersions[browserVersion])
     {
         throw new Error(
-            `Failed to locate a version of ${driverName} that matches the installed version of ${browserName} (${browserVersion}). Valid ${browserName} versions are: ${Object.keys(browserDriverVersions).join(', ')}`
+            `failed to locate a version of ${driverName} that matches the installed version of ${browserName} (${browserVersion}). Valid ${browserName} versions are: ${Object.keys(browserDriverVersions).join(', ')}`
```
Comment (thehcma) [inline installer.js]: ...of the <blah> driver that...

**Review by thehcma — APPROVED** [review]

**Comment by thehcma** [comment]
Once you fix all of the verbiage, please kick up the version.

**Comment by thehcma** [comment]
Please address the test failures.

**Review by thehcma — CHANGES_REQUESTED** [review]
relevant test failures - version needs updating

**Review by el — DISMISSED** [review]

File: installer.js (hunk 1 above)
Comment (sabgit) [reply installer.js]: fixed

File: installer.js (hunk 2 above)
Comment (sabgit) [reply installer.js]: fixed

File: installer.js (hunk 3 above)
Comment (sabgit) [reply installer.js]: fixed

File: installer.js [hunk 4]
```diff
@@ -123,7 +123,7 @@ function driverInstaller(browserName, browserVersion, targetPath)
     else
     {
         throw new Error(
-            `Browser name "${browserName}" is not a valid browser name. Valid browser names are: ${(VALID_BROWSER_NAMES).join(', ')}`
+            `browser name "${browserName}" is not a valid browser name. The valid names are: ${(VALID_BROWSER_NAMES).join(', ')}`
```
Comment (thehcma) [inline installer.js:126]: blah is not a valid browser name...

File: installer.js [hunk 5]
```diff
@@ -132,7 +132,7 @@ function driverInstaller(browserName, browserVersion, targetPath)
     if (browserVersion && !browserDriverVersions[browserVersion])
     {
         throw new Error(
-            `Failed to locate a version of ${driverName} that matches the installed version of ${browserName} (${browserVersion}). Valid ${browserName} versions are: ${Object.keys(browserDriverVersions).join(', ')}`
+            `failed to locate a version of the ${driverName} that matches the installed version of ${browserName} (${browserVersion}). Valid ${browserName} versions are: ${Object.keys(browserDriverVersions).join(', ')}`
```
Comment (thehcma) [inline installer.js:135]: ...the installed blah version

File: installer.js [hunk 6]
```diff
@@ -35,12 +35,12 @@ function installDriverWithVersion(driverName, driverBinPath, targetPath, version
...
             shell.cp('-n', path.join(TEMP_DIR, driverBinPath), targetPath);
             shell.rm('-rf', TEMP_DIR);
-            console.log('package dependencies have been installed');
+            console.log('the package dependencies have been installed');
             return true;
         },
         function (e)
         {
-            throw new Error('package dependencies installation failed with error, details: ' + e.toString());
+            throw new Error('the installation of the package dependencies has failed, details: ' + e.toString());
```
Comment (thehcma) [inline installer.js:43]: has failed -> failed (we use Simple Past always)

**Review by thehcma — APPROVED** [review]

**Pull request merged**